packages/doc-internal/package.json

```diff
@@ -4,10 +4,10 @@
 	"description": "Package to generate doc for other @huggingface packages",
 	"private": true,
 	"scripts": {
-		"start": "npm run doc-hub && npm run doc-inference && cp ../../README.md ../../docs/index.md && npm run update-toc && npm run fix-md-links",
+		"start": "pnpm doc-hub && pnpm doc-inference && cp ../../README.md ../../docs/index.md && pnpm update-toc && pnpm fix-md-links",
 		"format": "prettier --write . && eslint --quiet --fix --ext .cjs,.ts .",
-		"prepublish-hub": "npm run doc-hub && cp ../../README.md ../../docs/index.md && npm run update-toc && npm run fix-md-links",
-		"prepublish-inference": "npm run doc-inference && cp ../../README.md ../../docs/index.md && npm run update-toc && npm run fix-md-links",
+		"prepublish-hub": "pnpm doc-hub && cp ../../README.md ../../docs/index.md && pnpm update-toc && pnpm fix-md-links",
+		"prepublish-inference": "pnpm doc-inference && cp ../../README.md ../../docs/index.md && pnpm update-toc && pnpm fix-md-links",
```

Comment:
I don't think we need to change `npm run` to `pnpm`. If an external contributor mistakenly uses `npm install` instead of `pnpm install`, they can at least run the commands.

Reply:
it's an anti-pattern to allow more than one package manager in the same repo. `pnpm` is a pretty good choice and easy to install. It also takes much less disk space and is much faster than `npm`.
The `pnpm` team has an official tool to specifically handle the scenario you mention: https://github.com/pnpm/only-allow

Reply:
Ok, sounds good. 

Can you open a separate PR with the command changes + the only-allow package? 

I'd like to keep each PR changes scoped to the PR, for easier review / minimal diffs / git history once the PR is merged.